aesara/tensor/rewriting/math.py

```diff
@@ -311,6 +311,35 @@ def local_exp_log(fgraph, node):
         return [exp(x)]
 
 
+@register_specialize
+@node_rewriter([Elemwise])
+def log_diff_exp(fgraph, node):
+    # Case for log(exp(a) - exp(b)) -> a + log1mexp(b - a)
+    x = node.inputs[0]
+
+    if not isinstance(node.op, Elemwise):
+        return
```

Comment:
```suggestion
```

These checks against the `tracks` objects (i.e. `[Elemwise]`) are now automatic in the objects constructed by `node_rewriter`.  Again, we still need to update the old rewrites to reflect this at some point.